src/main/resources/logback-includes.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<included>
```

Comment:
Do we require to add this file? we can specify in application.yml file e.g
logging.level:
  org.springframework.web: ${LOG_LEVEL_SPRING_WEB:INFO}

Reply:
👍